core/src/main/resources/jenkins/model/Jenkins/login.jelly

```diff
@@ -24,10 +24,11 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <j:if test="${it.setupWizard != null}">
+  <j:choose>
+  <j:when test="${it.setupWizard.useGeneratedPassword()}">
```

Comment:
So `it.setupWizard` is now unconditionally nonnull? I did not see that change made on the Java side.